cscs-checks/microbenchmarks/kernel_latency/kernel_latency.py

```diff
@@ -75,10 +74,10 @@ def __init__(self, kernel_version):
             },
             'async': {
                 'dom:gpu': {
-                    'latency': (2.2, None, 0.10, 'us')
+                    'latency': (2.2, None, 0.15, 'us')
```

Comment:
From my experiments, I think that the reference should change and we might need to investigate.